package.json

```diff
@@ -24,12 +24,12 @@
     "shared/*"
   ],
   "scripts": {
+    "clean": "lerna run clean --parallel --scope @mux-elements/*",
     "format": "prettier --write .",
     "lint": "lerna run lint --scope @mux-elements/*",
     "test": "lerna run test --scope @mux-elements/*",
     "dev": "lerna run dev --parallel --scope @mux-elements/*",
     "build": "lerna run build",
```

Comment:
should we add `"prebuild": "clean"`

Reply:
Yeah it's out of scope but I am down. Gonna refactor my last version to just rely on your proposed prebuild as well.